extensions/wordpress-docs/src/index.tsx

```diff
@@ -84,8 +84,62 @@ function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
       actions={
         <ActionPanel>
           <ActionPanel.Section>
-            <Action.OpenInBrowser title="Open in Browser" url={searchResult.url} />
-            <Action.CopyToClipboard title="Copy URL to Clipboard" content={searchResult.url} />
+            <DefaultActions searchResult={searchResult} />
+            {environment.canAccess(AI) && (
+              <Action.Push
+                title="View Summary"
+                icon={Icon.Paragraph}
+                shortcut={{ modifiers: ["cmd", "shift"], key: "enter" }}
+                target={<Summary searchResult={searchResult} />}
+              />
+            )}
+          </ActionPanel.Section>
+        </ActionPanel>
+      }
+    />
+  );
+}
+
+function DefaultActions({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <>
+      <Action.OpenInBrowser title="Open in Browser" url={searchResult.url} />
+      <Action.CopyToClipboard title="Copy URL to Clipboard" content={searchResult.url} />
+    </>
+  );
+}
+
+function Summary({ searchResult }: { searchResult: SearchResult }) {
+  const item = JSON.stringify(searchResult);
+  const prompt = `Summarize the following from the WordPress docs and give one example of usage in a code block. The context can only be about WordPress. Format the response as if you are providing documentation:\n${item}`;
```

Comment:
Should we say `from the WordPress documentation`?

Reply:
Yeah I updated that now. 

I wonder whether one or the other has an effect on the quality of the output.